### ISSUE TITLE
Suggestion: More zoom options ?

### ISSUE THREAD

**Issue by dhuzard** [olivierfriard/BORIS#591]
It would be really useful if the zooming capacity could be slightly improved:
- To have a way to move the video once the video is moved ?
- Having more zooming options ?
(I'm doing behavior on mive, with camera view from the top, with different experimental cages in one view, so zooming helps a lot!)

ideally, a Crop-and-zoom in which we could manually crop the cage or videos zone that we want to see (thus cropping the other useless cages from the video).

Thanks! And thank you so so much for Boris!! 
Damien

**Comment by mbonneau13** [comment]
Yes, agree! I'm using MAc and Windows. On Mac OS, you can click on an area of interest of the image and it zooms to it. Very practicle. But not available on Windows.

**Comment by olivierfriard** [comment]
Hi,
the zoom function of v.8  (based on the MPV player) is not implemented exactly as in v.7 (based on the VLC player and no more developed).
I am working on this and I plan to release this function in a near future.
See  http://www.boris.unito.it/pages/boris_v8.html for the differences between v.7 and v.8

**Comment by dhuzard** [comment]
Thank you for your feedback! I'm using windows and V8.
Thus, I'm looking forward to the next releases :) !

**Comment by olivierfriard** [comment]
Hi Damien,
I  improved the zoom function in v. 8.17 (https://github.com/olivierfriard/BORIS/releases/tag/v8.16.7).

Now you can:
zoom in with `CTRL` + `+`
zoom out with `CTRL` + `-`
set no zoom with `CTRL` + `0` or by clicking the mouse right button on the video

and move (pan) the picture in video using:
`CTRL` + `LEFT ARROW`, `CTRL` + `RIGHT ARROW`, `CTRL` + `UP ARROW` and `CTRL` + `DOWN ARROW`, 

If you have n observation with many media you will have to select the video by a left-click before.

Please try it and let me know if you have suggestions or bugs to report
Thank you for your collaboration

**Comment by dhuzard** [comment]
Thank you so much, it is perfect !!
(just a quick comment, the moving commands are inverted (for example `CTRL` + `LEFT ARROW` goes Right). It is REALLY not a big deal ;) just to let you know)
thank you again! It helps so much!!
